marklogic-data-hub-spark-connector/src/main/resources/marklogic-data-hub-spark-connector/bulkIngester.api

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
This file should be inside datahub/5/modules according to the previous comment.

Reply:
It should be here - we want to move these modules to this location so that they can be inserted by a data-hub-operator user.

Reply:
Do we intend to maintain two copies of the same endpoint?

Reply:
I'm trying out the PR locally now - I'll post something to include in the connector's build.gradle file to copy the modules from the core project into the jar when the jar is built. That way, all the modules can live in the same place still. Currently, BulkIngestTest is failing because its modules no longer exist.

Reply:
I added this PR to copy the modules from the core project - https://github.com/anu3990/marklogic-data-hub/pull/4/files

Reply:
Included all the changes. Also rebased the code.